docs/apis-tools/frontend-development/introduction-to-task-applications.md

```diff
@@ -0,0 +1,141 @@
+
+Learn more how to work with the task queue in the [Tasklist user guide](/components/tasklist/userguide/using-tasklist.md).
+
+#### Task details panel
+
+The **task details panel** shows the details of the selected task. A [form](/guides/utilizing-forms.md) is displayed in the center of the details panel, which must be filled out to complete a task.
+
+:::tip
+Typically, a task application utilizes forms to capture information from the user, to make a decision, to collect the results from a real-world task, or to provide task instructions to the user.
+
+However, a [user task](/components/modeler/bpmn/user-tasks/user-tasks.md/#user-task-forms) is not limited to forms. A user task could also represent navigating to an external desktop or web application, where a task is to be performed, such as updating a record in a CRM. You can even use them to track physical work or actions using sensors, IoT devices, or any interface that can talk to the web, by registering a [job worker](/components/concepts/job-workers.md) to the job type `io.camunda.zeebe:userTask`.
+
+For these cases, utilize the flexible [custom form key](/components/modeler/web-modeler/advanced-modeling/form-linking.md/#custom-form-key).
+:::
+
+On the top of the form, a header shows the title of the task to work on, and the current assignee. Depending on the status of the assignment, a button allows you to assign the task to yourself or release it to the queue.
+
+At the bottom of the form there is a button with which you can complete the task.
+
+To the right of the task, you find additional information about the task, such as the [due date](/components/modeler/bpmn/user-tasks/user-tasks.md#scheduling) of the task, or the [user groups that can work on this task](/self-managed/concepts/access-control/user-task-access-restrictions.md).
+
+Potential extensions are dependent on your use case. You can consider adding more buttons to the bottom of the panel to indicate different task outcomes such as "approve" or "reject", or you could add a list of attachments to the right panel.
+
+Learn more how to work with the task details panel in the [Tasklist user guide](/components/tasklist/userguide/using-tasklist.md).
+
+### Processes page
+
+The **Processes** page lists all processes available to the logged in user, and allows the user to start a process from there.
+
+<img src={TasklistProcessesImg} style={{width: 800}} alt="Processes page in Tasklist" />
+
+Potential extensions are dependent on your use case. You can consider grouping processes by apps, domains, or teams, showing a process history, or adding a list of open process instances or cases.
+
+Learn more about the **Processes** page in the [Tasklist documentation](/components/tasklist/userguide/starting-processes.md).
+
+<!-- TODO move this to "concepts" -->
```

Comment:
Is this ready for review? I see a TODO HTML comment.

Reply:
The TODO is for a follow-up PR. Should we remove them from the HTML source? Are comments output in the built HTML?

Reply:
They should not output, but I just want to ensure someone will follow up and not forget about them.